RecFCCeeCalorimeter/src/components/CaloTowerToolFCCee.cpp

```diff
@@ -37,69 +43,70 @@ StatusCode CaloTowerToolFCCee::initialize() {
     return StatusCode::FAILURE;
   }
 
-  // check if readouts exist & retrieve Module-Theta segmentations
-  // if readout does not exist, reconstruction without this calorimeter part will be performed
-  std::pair<dd4hep::DDSegmentation::Segmentation*, SegmentationType> tmpPair;
+    std::vector<double> listPhiMax;
+  std::vector<double> listThetaMax;
+  listPhiMax.reserve(7);
+  listThetaMax.reserve(7);
```

Comment:
note for myself: we might want to refactor this tool in a later PR so that it accepts a list of cell collections and a list of readout names, making it more agnostic of the number of calorimeter sub detectors... 
Then this hardcoded 7 numbers would be the length of the list